ja/intro.rst

```diff
@@ -1,16 +1,140 @@
+CakePHP 概要
 ###################
 
-.. note::
-    The documentation is not currently supported in Japanese language for this
-    page.
+CakePHPはウェブ開発を単純に簡単にできるように開発されました。 オールインワンの
+ツールボックスは色々なパーツが一緒に動いたり、バラバラに動いたりできるようにします。
 
-    Please feel free to send us a pull request on
-    `Github <https://github.com/cakephp/docs>`_ or use the **Improve This Doc**
-    button to directly propose your changes.
+この概要は、一般的なコンセプトとそのコンセプトがどのように CakePHP の中で働くのかを紹介することです。
+プロジェクトをすぐに始めたいなら、 :doc:`チュートリアルから始める </tutorials-and-examples/bookmarks/intro>`か
+:doc:`直接ドキュメントを見て下さい</topics>`。
 
-    You can referer to the english version in the select top menu to have
-    information about this page's topic.
+設定に影響する規約
+==============================
+
+CakePHPは基礎的な構造をクラス名、ファイル名、DBのテーブル名や他の規約から決定します。
+規約を学ぶことで、不必要な設定や他の一般的なアプリと同じ構造をいちいち書かなくて済むので、
+簡単に色々なプロジェクトを進められます。この :doc:`規約</intro/conventions>` は、
+いろいろな CakePHP で使う規約をカバーしています。
+
+モデル(*Model*)層(レイヤー）
+===========================
+
+モデル層はビジネスロジックを実装するアプリケーションの部品を表します。これはデータの検索、アプリケーションに意味のある形への変換、また処理、検証（*validating*）、関連(*associating*)、そしてデータを扱うことに関する様々なタスクに責任をもつことを意味します。
+
+一見して、モデルオブジェクトはアプリケーションに使用しているであろうデータベースとやりとりする最初の層と見ることができるでしょう。
+しかし、一般的にこれはアプリケーションを実装するものの主要な概念を表します。
+
+ソーシャルネットワークのケースでは、モデル層はユーザのデータを保存する、友人の繋がりを保存する、ユーザの写真を保存または検索する、新しい友人の提案を検索する、等々のタスクを引き受けることでしょう。
+このとき、モデルオブジェクトは「友達(*Friend*)」、「ユーザ(*User*)」、「コメント(*Comment*)」、「写真(*Photo*)」と考えることができます。
+``users`` テーブルからデータを読みたかったら、::
+
+    use Cake\ORM\TableRegistry;
+
+    $users = TableRegistry::get('Users');
+    $query = $users->find();
+    foreach ($query as $row) {
+        echo $row->username;
+    }
+
+データを使って仕事をする前に一切コードを必ず書かなくてもよいことに気づくでしょう。
+規約を使うことによって、CakePHP はまだ定義されていないテーブルとエンティティクラスのためのスタンダードクラスを使うようになります。
+
+新しいユーザーを作ってバリデーションしてから保存したいなら、このようにします::
+
+    use Cake\ORM\TableRegistry;
+
+    $users = TableRegistry::get('Users');
+    $user = $users->newEntity(['email' => 'mark@example.com']);
+    $users->save($user);
+
+ビュー(*View*)層(レイヤー）
+==========================
+
+ビューレイヤーは、モデルから来たデータをレンダリングします。ビューはモデルオブジェクトとは別に存在します。
+そして、扱っている情報に対してレスポンシブルなアプリケーションが必要としている表示インターフェイスをすべて提供可能です。
+
+例えば、このビューはHTMLかXMLとして他人が消費するための結果をモデルから送られてくるデータを利用してレンダリングできます。::
+
+    // ビューファイルで 'element' をそれぞれのユーザーに対してレンダリングする
+    <?php foreach ($users as $user): ?>
+        <div class="user">
+            <?= $this->element('user', ['user' => $user]) ?>
+        </div>
+    <?php endforeach; ?>
+
+このビューレイヤーは :ref:`view-elements` や :doc:`/views/cells` のようなしくみで表示のためのロジックを再利用可能にして、
+沢山の表示を拡張するための機能を提供します。
+
+ビューレイヤーはHTMLやテキストのレンダリングを制御出来るだけではなく、一般的なJSONやXML、
+加えてプラグインで追加可能なアーキテクチャによるフォーマットなら何にでも対応します。
+
+
+コントローラ(*Controller*)層(レイヤー）
+======================================
+
+コントローラ層はユーザからのリクエストを扱います。
+これはモデル層とビュー層の助けを借りてレスポンスをレンダリングして返す責任を負います。
+
+コントローラは、タスクを終える為の全ての必要とされるリソースが正しい労働者に委譲されることに注意を払うマネージャーと見ることができます。
+クライアントからの要求を待ち、認証と承認のルールによる検証を行い、データの取得または処理をモデルに委譲し、クライアントが受け入れる適切な表示上のデータの種類を採択し、最終的にその描画処理をビュー層に委譲します。
+例えば、ユーザー登録ではこのようになります。:
```

Comment:
in tow byte letter it need only single : .
:: express : in view but ; is not in Japanese but many Japanese can guess this meaning.
Because I think  :: or : both of it is OK/